YOU MAY GET AN ERROR AT FIRST TIME YOU RUN THE CODE SAYING: IndexError: tuple index out of range

SOLUTION: RUN THE CODE AGAIN! IT'S JUST A NETWORK ISSUE IN DATA RETRIEVAL FROM chEMBL!
